src/Bridges/quadtosocbridge.jl

```diff
@@ -86,8 +87,9 @@ function QuadtoSOCBridge{T}(model, func::MOI.ScalarQuadraticFunction{T},
                   " not quadratic constraints but you entered a quadratic",
                   " constraint of type: `$(typeof(func))`-in-`$(typeof(set))`.",
                   " A bridge attempted to transform the quadratic constraint",
-                  " to a second order cone constraint but the constraint is not",
-                  " convex.")
+                  " to a second order cone constraint but the constraint is",
+                  " not strongly convex, i.e., the symmetric matrix of"
+                  " quadratic coefficients is not positive definite.")
```

Comment:
This wording suggests that the user did something wrong, while the issue is in the implementation of the bridge instead. Maybe "... not yet supported" is more appropriate.